src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationSettingTests.cs

```diff
@@ -104,6 +104,40 @@ public void FilterOnlyLabel()
             Assert.AreEqual($"http://localhost/?key=*&label={label}", builder.Uri.AbsoluteUri);
         }
 
+        [Test]
+        public void SettingSomeFields()
+        {
+            var service = new ConfigurationClient(s_connectionString);
+
+            var selector = new SettingSelector("key")
+            {
+                Fields = SettingFields.Key | SettingFields.Value
+            };
+
+            var builder = new HttpPipelineUriBuilder();
+            builder.Uri = new Uri("http://localhost/");
+            service.BuildBatchQuery(builder, selector);
+
+            Assert.AreEqual($"http://localhost/?key=key&$select=key,%20value", builder.Uri.AbsoluteUri);
```

Comment:
Should we remove the space from concatenation? I don't think it matters and it would make URIs shorter.

Reply:
Talked offline about this. To remove the extra space we will have to overwrite the `ToString` or do a `.Replace` as by default the `Enum.ToString()` adds the space.
We decided to leave it as it is.